Assets/Mirror/Tests/Editor/WeaverAssembler.cs

```diff
@@ -1,14 +1,38 @@
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Collections.Generic;
+using UnityEditor;
 using UnityEditor.Compilation;
 using UnityEngine;
 
 namespace Mirror.Tests
 {
     public class WeaverAssembler : MonoBehaviour
     {
-        public const string OutputDirectory = "Assets/Mirror/Tests/Editor/WeaverTests~/";
+        static string _outputDirectory;
+        public static string OutputDirectory
+        {
+            get
+            {
+                if (string.IsNullOrEmpty(_outputDirectory))
+                {
+                    string[] guidsFound = AssetDatabase.FindAssets($"t:Script WeaverAssembler");
```

Comment:
how about typeof(WeaverAssembler) so we don't hardcode the class name?

Reply:
I will change it to nameof(WeaverAssembler)